stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -293,6 +293,9 @@ spec:
             initialDelaySeconds: 30
             periodSeconds: 10
 {{ end }}
+        {{- if .Values.sidecarContainers }}
+        {{- toYaml .Values.sidecarContainers | nindent 8 }}
```

Comment:
This should probably be ran through tpl first: `{{ tpl (toYaml .Values.extraContainers) . | nindent 8 }}`